testCourse/questions/barcodeSubmission/server.py

```diff
@@ -0,0 +1,4 @@
+import random, copy
+
+def generate(data):
+    return data
```

Comment:
You should just be able to completely omit `server.py` instead of including this no-op.

Reply:
Done.